Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/View.purs

```diff
@@ -2936,6 +2936,10 @@ getEstimate action flowStatusAction count duration push state = do
           _ <- pure $ printLog "api Results " response
           let (GetQuotesRes resp) = response
           if not (null resp.quotes) || not (null resp.estimates) then do
+            if not (null resp.quotes) then do
```

Comment:
we can use the fare product type if we need it specifically for special zone.

Reply:
So whenever we get quotes we can get this log, better to print in all the cases when the quotes come

Reply:
then we can change the logEvent name to `ny_user_quotes`

Reply:
`ny_user_quotes` We are already using somewhere have added an alternative